msu/utils/text.nut

```diff
@@ -28,4 +28,97 @@
 	{
 		return this.color(::Const.UI.Color.NegativeValue, _string);
 	}
+
+	function colorizeValue( _value, _kwargs = null )
+	{
+		local kwargs = {
+			AddSign = true,
```

Comment:
Should this default to `true` or `false`? Do you by default always want a sign attached? How has been the experience using this function in Reforged?

Reply:
I've found that in most of our use cases we didn't want the sign attached, so by default I think I'll make it `false`.